core/src/main/java/org/apache/iceberg/SchemaParser.java

```diff
@@ -183,7 +183,6 @@ public static String toJson(Schema schema, boolean pretty) {
   private static Type typeFromJson(JsonNode json) {
     if (json.isTextual()) {
       return Types.fromPrimitiveString(json.asText());
-
```

Comment:
Nit: unnecessary whitespace change.

Reply:
yes I know but I didn't feel it's worth opening up a separate PR to remove this whitespace line

Reply:
Why remove it at all? Seems like an unncessary change.

Reply:
I stumbled over it already a couple of times and figured I just remove this unnecessary line ;)